ChemistryLib/PhreeqcIOData/Output.h

```diff
@@ -24,7 +24,7 @@ class BasicOutputSetups
 public:
     explicit BasicOutputSetups(std::string const& project_file_name,
                                bool const use_high_precision_)
-        : output_file(project_file_name + "_phreeqc.out"),
```

Comment:
This must not be changed.

Reply:
Checked by 
1. finding all strings and dump the results to file by the command of
```
 grep -R '"'  -r -i --include \*.h --include \*.cpp > comp.txt
```
2. comparing the file by ufz/master and the branch of this PR. In addition to the one you found, there is another in PhreeqcIO.cpp.  These two changes were just dropped.